zebedee-cms/src/main/java/com/github/onsdigital/zebedee/session/service/SessionsServiceImpl.java

```diff
@@ -124,9 +128,13 @@ public Session get(String id) throws IOException {
             // Deserialise the json:
             Session session = sessionsStore.read(sessionPath(id));
             if (!expired(session)) {
+                info().log("Session running successfully during session get");
```

Comment:
log events should be lower case (https://github.com/ONSdigital/dp/blob/main/standards/LOGGING_STANDARDS.md#effective-event-types)

Reply:
This has now been amended